compiled_starters/typescript/app/main.ts

```diff
@@ -0,0 +1,27 @@
+import * as dgram from "dgram";
+
+// You can use print statements as follows for debugging, they'll be visible when running tests.
+console.log("Logs from your program will appear here!");
+
+// Uncomment this block to pass the first stage
+//
+// const udpSocket: dgram.Socket = dgram.createSocket("udp4");
+// udpSocket.bind(2053, "127.0.0.1");
+
+udpSocket.on("message", (data: Buffer, remoteAddr: dgram.RemoteInfo) => {
+    try {
```

Comment:
@ryan-gang let's make this simpler: 

(a) Do we need a "message" callback? If not, let's remove this
(b) If we need the message callback to keep the server alive, then let's just print out something like "Received data from ${remoteAddr}" here.